sssom/parsers.py

```diff
@@ -540,8 +547,14 @@ def from_alignment_minidom(
                 elif node_name == "onto2":
                     ms[OBJECT_SOURCE_ID] = e.firstChild.nodeValue
                 elif node_name == "uri1":
-                    ms[SUBJECT_SOURCE] = e.firstChild.nodeValue
+                    # import pdb; pdb.set_trace()
+                    ms[
+                        SUBJECT_SOURCE
+                    ] = (
+                        e.firstChild.nodeValue
+                    )  # CURIEfy this node value && If endswith(".extension"), replace it with _)
```

Comment:
What is your plan here re curification if an URI prefix is not in the prefix map?

Reply:
I would vote for leaving the URIs as is if the URI prefix is not known to the prefix map. We should produce a single warning at the end of the pipeline which lists the first 5 URIs that didn't have a proper CURIE, and log a warning "the extracted file is not valid SSSOM because some of the URIs could not be converted. Please provide appropriate URI prefixes and repeat  the process".


Charlie prefers to not process the node at all if the URI prefix is unknown; this is an alternative, but only if we can overwrite it with a flag, and then we are back to the first point.

Reply:
That is the problem. The `sssom` definition for `subject_source` and `object_source` is `EntityReference`. Does this mean only CURIEs? `uriOrCurie` might be a better solution?

Reply:
I think a URI _is_ a curie, where the prefix is `http` and the URI expansion `http` so everwhere where there is an entity reference, you should be able to drop a URI.

Its fine to write a URI, just also write the warning I mention above.